app/data/contentscript.js

```diff
@@ -3,31 +3,31 @@
 
 var pathComponents = window.location.pathname.split('/');
 var _logName = 'github-forks-extension:';
-var lovelyForksClass = 'lovely-forks-addon-class';
 var DEBUG = false;
+var el = new WeakMap();
```

Comment:
WeakMap will not hold the element when GH removes the element (on ajax navigation, for example) and the browser does GC. 

WeakMap has been there since Firefox 6 ;)